contracts/common/MetaTransactionMsgSender.sol

```diff
@@ -25,4 +25,17 @@ abstract contract MetaTransactionMsgSender is DSMath {
       return payable(msg.sender);
     }
   }
+
+  function isMetatransaction() internal view returns (bool) {
```

Comment:
Is it possible to use this new `isMetatransaction` function as part of `msgSender()`? Otherwise there is significant code duplication in this fiel.

Reply:
That's fair; please review that refactorisation carefully though :smile: